pom.xml

```diff
@@ -242,6 +242,7 @@
         <module>service-discovery/consul</module>
         <module>service-discovery/kubernetes</module>
         <module>service-discovery/eureka</module>
+        <module>service-discovery/combined</module>
```

Comment:
I would have named it composite and not combined. What leads you to combine?

Reply:
it combines results from other ones :)
composite is good for me too, I'll rename it

Reply:
We should rename all the current `Combined` stuff, no? the classes....

Reply:
yes, that's what I'm doing in an incoming rework :)